fs/ignorefs/ignorefs.go

```diff
@@ -25,22 +25,29 @@ type ignoreContext struct {
 
 	onIgnore []IgnoreCallback
 
-	dotIgnoreFiles []string         // which files to look for more ignore rules
-	matchers       []ignore.Matcher // current set of rules to ignore files
-	maxFileSize    int64            // maximum size of file allowed
+	dotIgnoreFiles []string                  // which files to look for more ignore rules
+	matchers       []wcmatch.WildcardMatcher // current set of rules to ignore files
```

Comment:
any reason for the new package? can we just repurpose the old one?

Reply:
There's a reason for everything in this world, the only question whether or not it is a good reason. 😉 

Jokes aside, I started developing this as a package outside of kopia, just for wildcard matching which is where the name came from. When I moved it in it seemed more descriptive to me which is why I kept it. The matcher does globbing or wildcard matching and really has no code for "ignoring" anything, this logic is all in `ignorefs`.

But I can of course change the package name back to "ignore" if you prefer this. Just let me know.  😃